### DISCUSSION TITLE
M1 Arm Version

### DISCUSSION THREAD

**Discussion by BicycleMark** [dotnet/maui#12951] [General]
I am running 

Microsoft Visual Studio Community 2022 (ARM 64-bit) - Current
Version 17.4.4

Is there a MAUI Workload available for this version of Visual Studio? I think not. If I am wrong please tell me how to load it. If it is not available then tell me when is it likely to be available?

Thanks in Advance!

**Comment by mphill** [comment]
Download VS Mac and reinstall. 

For the record VS Mac is very neglected compared it it’s windows counter part. It didn’t get maui support until recently so if you had it install for a while this could explain why it’s missing

**Comment by davidortinau** [comment]
.NET MAUI workloads with arm support for Visual Studio 2022 Windows is coming very soon. I'm previewing it now. Look for it in the preview channel in coming weeks.
